### PR TITLE
Create hiring-work-for-users.md

### PR DESCRIPTION
**Pull request by harryharrold**
Fixes gap in our staffing

## Description

- recruitment post
-

### CODE REVIEW

File: src/data/blog/hiring-work-for-users.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+slug: 'hiring-work-for-users'
+image: null
+featured: 0
+page: 0
+status: 'published'
+language: 'en_US'
+meta_title: null
+meta_description: null
+author_id: 1
+created_at: '2021-07-09T13:29:43.000Z'
```
Comment (Nikomus) [inline src/data/blog/hiring-work-for-users.md:12]: I think this needs a published_at as well, otherwise it defaults to 01-01-1970

Comment (harryharrold) [reply src/data/blog/hiring-work-for-users.md:12]: Neat - added that in. Thx.

**Comment by harryharrold** [comment]
@Nikomus Published at added.

**Review by Nikomus — APPROVED** [review]
The title is repeated in the first section heading but otherwise looks good. The link is here for anyone else having a look:
https://neontribe-www-git-harryharrold-recruitment-2-neontribe.vercel.app/blog/hiring-work-for-users

**Review by edpe — APPROVED** [review]

**Pull request merged**